docs/installing-solidity.rst

```diff
@@ -118,8 +118,24 @@ The nightly version can be installed using these commands:
     sudo apt-get update
     sudo apt-get install solc
 
-We are also releasing a `snap package <https://snapcraft.io/>`_, which is
-installable in all the `supported Linux distros <https://snapcraft.io/docs/core/install>`_. To
+Furthermore, some Linux distributions provide their own packages. These packages are not directly
+maintained by us, but usually kept up-to-date by the respective package maintainers.
+
+Arch Linux has packages for the latest development version:
+
+.. code-block:: bash
+
+    pacman -S solidity
+
+Gentoo Linux has an `Ethereum overlay <https://overlays.gentoo.org/#ethereum>`_ that contains a Solidity package.
```

Comment:
https://github.com/coolparadox/ethereum-gentoo-overlay/tree/master/dev-lang/solidity actually doesn't look particularly up-to-date :-)... I'll just keep it in here for now, but we should probably do something about it as well (i.e. either poke the maintainers or mention that it's falling out of date here or remove it)...

Reply:
Maybe it would be best that it's unmaintained just like the snap? I'd either do that or just drop it.

Reply:
Hm, yeah... I think I'll drop it and create an issue in the repo there, letting them know that we'll add it back, if they update it - does that sound reasonable :-)?

Reply:
https://github.com/coolparadox/ethereum-gentoo-overlay/issues/1

Reply:
Sounds good.